Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.11.6-alpine3.18
+LABEL maintainer="savenya2013@gmail.com"
+
+ENV PYTHONDONTWRITECODE=1
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY app/ ./app
```

Comment:
The `COPY app/ ./app` command assumes that there is an `app` directory at the same level as the Dockerfile. Ensure that this directory exists and contains the necessary files.